<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundingBox</class>
 <widget class="QWidget" name="BoundingBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>186</width>
     <height>68</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>186</width>
     <height>68</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>186</width>
     <height>68</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QDoubleSpinBox" name="northDSpinBox">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>4</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-90.000000000000000</double>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>6</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>26</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>46</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="eastDSpinBox">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>24</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>26</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="southDSpinBox">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>44</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-90.000000000000000</double>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="westDSpinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>20</height>
     </size>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
